Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
     dependencies: [
         // Help for the format of declaring SPM dependencies:
         // https://web.archive.org/web/20220525200227/https://www.timc.dev/posts/understanding-swift-packages/
-        .package(name: "Gist", url: "https://gitlab.com/bourbonltd/gist-apple.git", from: "2.2.1"),
+        .package(name: "Gist", url: "https://github.com/customerio/gist-apple.git", from: "2.2.1"),
```

Comment:
```suggestion
        .package(name: "Gist", url: "https://github.com/customerio/gist-apple.git", from: "2.2.4"),
```

Since 2.2.4 is the latest version, should this be changed? 

---

Also, since this PR is changing the Gist SDK version to 2.2.4 for SPM, should this PR also install 2.2.4 for Cocoapods?